### ISSUE TITLE
[NTR] Add pulmonary ionocyte CL term

### ISSUE THREAD

**Issue by mshadbolt** [obophenotype/cell-ontology-training#6]
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
pulmonary ionocyte

**Synonyms**
lung ionocyte

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
An ionocyte that is part of a lung which co-expresses FOXI1, multiple subunits of the vacuolar-type H+-ATPase (V-ATPase) and CFTR
PMID:30069046 

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
epithelial cell of lung http://purl.obolibrary.org/obo/CL_0000082

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
part of lung epithelium http://purl.obolibrary.org/obo/UBERON_0000115

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-7505-5418

**Any additional notes or concerns**
I would like to add links to the genes expressed in the paper i.e.
FOXI1
V-ATPase
CFTR

**Comment by dosumis** [comment]
We already have this: http://purl.obolibrary.org/obo/CL_0017000

**Comment by dosumis** [comment]
definition "An ionocyte that is part of the lung epithelium. The cells from this type are major sources of the CFTR protein in human and mice."

**Comment by dosumis** [comment]
We could potentially add markers - although not directly to a species neutral term.  I assume these are for human.

**Comment by lubianat** [comment]
That was actually my first CL cell type :) 
The original articles characterized the cell type both in mice and humans. 
In the original articles, they kind of mention the "CFTR gene" in a semi species neutral way, but I was not sure how to add the markers.

**Comment by lubianat** [comment]
I did not put "lung ionocyte" as a synonym, though, that could be readily updated (I think)

**Comment by mshadbolt** [comment]
Yeah I knew it was already in the ontology, I was just practicing adding a request for the training session. Though I do think it might be cool to add the marker genes if the evidence in the paper is considered good enough to do this.

I have kind of skimmed the paper and I think this cell type has specifically only been found in mice and human, but seems similar to the ionocytes in gills of fish. I wonder if there is any way of characterising that kind of relationship in the ontology?

**Comment by mshadbolt** [comment]
> I did not put "lung ionocyte" as a synonym, though, that could be readily updated (I think)

I also kind of made that up as a synonym, just to practice, I don't know if anyone uses that as a synonym...
